README.md

```diff
@@ -1,2 +1,4 @@
 # dterm
 💻 Interact with toy terminal instance from Discord with global state atomically stored in Deno Kv. 
+
+This GitHub issue proposes the development of a Discord bot that incorporates the usage of the headless version of xterm.js1. The bot will enhance traditional bot functionality by integrating a virtual terminal environment, allowing users to execute supported CLI commands and receive outputs within Discord.
```

Comment:
Nit: Stray `1` in the paragraph.

Reply:
Resolved in <https://github.com/acmcsufoss/dterm/pull/2/commits/b5a2125e2348f9b3b3f30cf3501d5991c35879a6>.